reana_client/cli/files.py

```diff
@@ -206,8 +206,8 @@ def download_files(ctx, workflow, filenames, output_directory, access_token):
 
 @click.command(
     'upload',
-    help='Upload all input sources declared in the reana.yaml'
-         'specification or upload files and directories listed as '
+    help='Upload workflow input files and sources. Files are declared in the '
```

Comment:
Upload workflow input files and directories.  The sources can be specified in the command line (see examples below).  The default behaviour is to upload all input files and directories specified in the `reana.yaml` file.

*(an example of a language we could use for all commands -- say what it does, what users can pass on, and what is happening by default, and finish by concrete examples people could copy/paste, as mentioned in the issue)*

Reply:
I like this simple phrasing, I would only simplify a bit more to
`[short description]. [by default behaviour with mention to the example below]. `
But honestly I have checked `git` and `kubectl` as examples and they use the `if not blahblah, command will blahblah` and  they are not necessary next to the short description, for instance in `kubectl describe --help` it is in the 2nd paragraph, see examples:
```
$ git checkout --help
... If no paths are given, git checkout will also update HEAD to
set the specified branch as the current branch. ...
$ kubectl describe --help
... If no such resource exists, it will output details for
every resource that has a name prefixed with NAME PREFIX. ...
```
If it doesn't make sense to you @tiborsimko @roksys just ignore my https://github.com/reanahub/reana-client/pull/227#discussion_r253765028 and related to default behaviour phrasing.